src/webview/create-component/pages/fromExistingGitRepo.tsx

```diff
@@ -453,7 +453,7 @@ export function FromExistingGitRepo({ setCurrentView }) {
                         setCurrentPage('fromGitRepo');
                     }}
                     createComponent={createComponentFromGitRepo}
-                    devfile={recommendedDevfile.isDevfileExistsInRepo ? undefined : selectedDevfile ? selectedDevfile : recommendedDevfile.devfile}
+                    devfile={recommendedDevfile.isDevfileExistsInRepo ? recommendedDevfile.devfile : selectedDevfile ? selectedDevfile : recommendedDevfile.devfile}
```

Comment:
It would be nice to pass a boolean indicating whether we are working with an existing Devfile, and disabling the component name and port fields in this case, since setting the component name and port will do nothing when there is an existing Devfile, which is unintuitive.

Reply:
Acknowledged